test/functional/test-layout.js

```diff
@@ -275,6 +275,40 @@ describe('Layout', () => {
     expect(div.children[0].style.paddingTop).to.equal('200%');
   });
 
+
+  it('layout=intrinsic', () => {
```

Comment:
visual tests for this would be great. please add to existing layout visual tests in `examples/visual-tests/amp-layout`

Reply:
will do